docs/data/sdks/ios-swift/index.md

```diff
@@ -58,7 +58,8 @@ let amplitude = Amplitude(configuration: Configuration(
     | `serverZone` |  The server zone to send to, will adjust server url based on this config. | `US` |
     | `useBatch` |  Whether to use batch api. | `false` |
     | `trackingOptions` |  Options to control the values tracked in SDK. | `enable` |
-    | `enableCoppaControl` |  Whether to enable COPPA control for tracking options. | `false` |'
+    | `enableCoppaControl` |  Whether to enable COPPA control for tracking options. | `false` |
+    | `migrateLegacyData` | `Boolean`. Available in `0.4.7`+. Whether to migrate [maintenance SDK](../ios) data (events, user/device ID). | `true`|
```

Comment:
Existing boolean options (`enableCoppaControl`, `useBatch`, etc) do not specify `Boolean`  type explicitly.

Reply:
Fixed. Removed the `Boolean` to match other fields.